subsys/sal/common/sid_ifc/bt_app_callbacks.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <bt_app_callbacks.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <stdbool.h>
+#include <zephyr/kernel.h>
+
+static uint32_t bt_enable_count = 0;
+
+int app_bt_enable(bt_ready_cb_t cb)
```

Comment:
In genreal, I would recommend to keep the naming convention of Sidewalk module. The `app_` prefix is used for sample configuration like `app_ble_config`. For ble code we used `sid_ble_` prefixes (`sid_ble_advert`, `sid_ble_connectopn`)